frontend/stories/pages/application.stories.tsx

```diff
@@ -0,0 +1,20 @@
+import { Meta } from "@storybook/react";
```

Comment:
I don't think we need to the "application story, should be able to access the opportunityCard story without it